english/bob-barker/2050s/2050s.md

```diff
@@ -14,4 +14,6 @@ Wear a black suit he told you, Maxwell hates blue suits and be sure to bring you
 
 Your iClock tells you its 3:30 PM.  'Ah, nostalgia', you think.  As time is everywhere and in everyone.
 
-A hushed voice whispers, "There can be only one."
\ No newline at end of file
+A hushed voice whispers, "There can be only one."
+
+[You turn around to find your quiet comrade.] (dragon-slave/dragon-slave.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.